### DISCUSSION TITLE
Fixing out of sync Interlisp.github.io fork

### DISCUSSION THREAD

**Discussion by pamoroso** [Interlisp/medley#1882] [General]
To keep my [Interlisp.github.io fork](https://github.com/pamoroso/Interlisp.github.io) synchronized with the original I use the `Sync fork` menu command on GitHub.

Today the fork was 2 commits ahead (already merged) and `Sync fork` provided two options, merging or removing the commits. I usually do the latter but today I picked the merge option, which resulted in the fork still being 2 commits ahead. Moreover, creating a PR included those commits.

In an attempt to bring the fork back in sync I submitted a [dummy PR](https://github.com/Interlisp/Interlisp.github.io/pull/251) asking @stumbo to reject it, which he did, in the hope GitHub would give again the syncing option to remove the commits. But the fork is still 2 commits ahead and now creating a PR includes the commits *and* lists the dummy PR:

![fork-pr](https://github.com/user-attachments/assets/1b11681b-3f92-4ab3-95d4-faef1082f9a7)

On GitHub I can't find any way of deleting or undoing the commits. How can I bring my fork back in sync?

**Comment by nbriggs** [comment]
Can you just delete the fork and re-fork it?

**Reply by pamoroso** [reply]
If I delete the fork GitHub deletes only my copy even if I'm a member of the Interlisp organization, right?

**Reply by masinter** [reply]
Right, deleting a fork doesn't affect the repo your fork was made from.

**Reply by pamoroso** [reply]
I deleted my fork and re-forked, now everything is fine again.

**Comment by MattHeffron** [comment]
@pamoroso OK to close this?

**Comment by pamoroso** [comment]
@MattHeffron Sure, I'm closing the discussion.
